pom.xml

```diff
@@ -37,6 +39,37 @@
                 <artifactId>debezium-connector-oracle</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Wouldn't it suffice to specify `${version.debezium}` in the reference to the _debezium-parent_ POM?